wagl/acquisition/sensors.json

```diff
@@ -521,6 +521,230 @@
             }
         }
     },
+    "LANDSAT_9": {
+        "OLI": {
+            "band_ids": {
+                "1": {
+                    "alias": "Coastal-Aerosol",
+                    "band_type": "REFLECTIVE",
+                    "band_name": "BAND-1",
+                    "wavelength": [0.433, 0.453],
+                    "spectral_filter_name": "landsat9_vsir.flt",
+                    "spectral_range": [2600, 349, -1],
+                    "reflectance_adjustment": 0.0365,
+                    "brdf_datasets": ["BRDF_Albedo_Parameters_Band3"],
+                    "supported_band": true
+                },
+                "2": {
+                    "alias": "Blue",
+                    "band_type": "REFLECTIVE",
+                    "band_name": "BAND-2",
+                    "wavelength": [0.45, 0.515],
+                    "spectral_filter_name": "landsat9_vsir.flt",
+                    "spectral_range": [2600, 349, -1],
+                    "reflectance_adjustment": 0.0365,
+                    "brdf_datasets": ["BRDF_Albedo_Parameters_Band3"],
+                    "supported_band": true
+                },
+                "3": {
+                    "alias": "Green",
+                    "band_type": "REFLECTIVE",
+                    "band_name": "BAND-3",
```

Comment:
I'd double-check with Mark and Fuqin on these values. They differ from [USGS' website](https://www.usgs.gov/landsat-missions/landsat-9)

Whilst I don't think they're used in any algorithmic sense (anymore), it's still ideal to have them correct.

Reply:
the wavelength ranges i mean